src/js/components/Ballot/OfficeItemCompressed.jsx

```diff
@@ -44,6 +46,11 @@ class OfficeItemCompressed extends Component {
     if (officeWeVoteId && !BallotStore.positionListHasBeenRetrievedOnce(officeWeVoteId)) {
       OfficeActions.positionListForBallotItemPublic(officeWeVoteId);
     }
+    const addressBlock = document.querySelector('.u-link-color.u-link-underline-on-hover')?.innerText;
```

Comment:
I want to avoid using style selectors -- we only use ids to refer to elements in the DOM. The State Code should be in the officeData (which you are using in the ShowMore component) so I think we can remove the retrieval of stateCode in the OfficeItemCompressed component.